### PR TITLE
officially adds the functionality to insert holdable mobs into colormate

### PR DESCRIPTION
**Pull request by silicons**
## About The Pull Request

the holder glitch without the glitch

## Why It's Good For The Game

it probably isn't but people kept yelling at me to readd it properly (read: someone screamed NOOOOOOO in OOC and I felt bad)

## Changelog
:cl:
add: colormates can now paint some mobs.
/:cl:

### CODE REVIEW

File: code/game/machinery/colormate.dm [hunk 1]
```diff
@@ -57,6 +59,15 @@
 		return
 	if(user.a_intent == INTENT_HARM)
 		return ..()
+	if(allow_mobs && istype(I, /obj/item/clothing/head/mob_holder))
+		var/obj/item/clothing/head/mob_holder/H = I
+		var/mob/victim = H.held_mob
```
Comment (ghost) [inline code/game/machinery/colormate.dm:64]: dumb of ass

Comment (ghost) [reply code/game/machinery/colormate.dm:64]: where's my server box access at

**Comment by BlueWildrose** [comment]
Blue Ian, here we come.

**Comment by YakumoChen** [comment]
Cargo nerf 👎 
Colored corgi is supposed to be special

**Pull request merged**